package.json

```diff
@@ -66,6 +66,7 @@
     "ethers": "^4.0.41",
     "eventemitter3": "^4.0.0",
     "heap": "^0.2.6",
+    "lodash.uniqueid": "^4.0.1",
```

Comment:
No need to do this any more, now you can depend on `lodash` directly and then do:

```js
import uniqueId from 'lodash/uniqueid'
```

 What you've got is fine for now, but there are many useful, time/code saving features in `lodash` so IMO better to just pull it in as a full dep so it's all readily available.